bundles/org.openhab.binding.digitalstrom/pom.xml

```diff
@@ -14,4 +14,12 @@
 
   <name>openHAB Add-ons :: Bundles :: DigitalSTROM Binding</name>
 
+  <dependencies>
+    <dependency>
+      <groupId>org.assertj</groupId>
```

Comment:
Is there any reason for using assertj instead of the provided assertions?

Reply:
There is no special reason. I'm used to assertJ and in my opinion, assertJ provides a much better fluent interface (and very clever assertions) than most assertion frameworks.
I can change this to the desired framework, if you wish.

Reply:
Yes please, it's always good to keep the external dependencies to a minimum.

Reply:
I've removed the assertJ dependency and I've rewritten the tests to hamcrest - I now remember why I switched to assertJ a few years ago ;)